src/languages/pyret/PyretParser.ts

```diff
@@ -319,10 +319,10 @@ const nodeTypes = {
 
   // data Program
   "s-program": function(_pos: Loc, _provides: ASTNode, _provTy: any, imports: ASTNode[], body: Block) {
-    console.log("Provides in program:", _provides.length)
+    console.log("Provides in program:", _provides);
```

Comment:
Thank you @RobScheidegger for answering my question here. I decided to just leave these debug logging calls in here in case someone else is expecting them. Would be great to find an alternative method for debugging in the future though.